lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+                    "code": "667687"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                }
+              ],
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED",
+              "code": "64897"
+            }
+          ]
+        }
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the CDM_SCHED, CPT, HCPCS, or PROCCODE proprietary Bill Codes Types systems.</li>
+        <li>Tiering logic from highest to lowest: CDM_SCHED, CPT, HCPCS, or PROCCODE</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code
+
+  - name: performer
+    required: 'No'
+    type: List of BackboneElement
+    description: Indicates who performed or participated in the charge.
+    action: modify
+    example: |
+      {
+        "performer": [
+          {
+            "function": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
+                  "code": "OP",
+                  "display": "Ordering Provider"
+                }
+              ],
+              "text": "Ordering Provider"
+            },
+            "actor": {
+              "reference": "Practitioner/11817978"
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Need notes around supported system and values for function</li>
+        <li>Both actor and function are required</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.performer
+
+  - name: reason
+    required: 'No'
+    type: CodeableConcept
+    description: Diagnosis codes for why the charge was rendered.
+    action: modify
+    example: |
+      {
+        "reason": [
+          {
+            "extension": [
+              {
+                "valueUnsignedInt": 1,
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+              }
+            ],
+            "coding": [
+              {
+                "extension": [
+                  {
+                    "valueCoding": {
+                      "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                      "code": "3693"
+                    },
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                  }
+                ],
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
+                "code": "R51"
+              }
+            ]
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the ICD proprietary Bill Codes Types systems.</li>
```

Comment:
We should ensure we are being consistent with our punctuation. I believe we did no '.' throughout credit, so let's continue that here.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/928dc349ae56b5c07470ad207306beef33bc2b8d#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R217